scipy/optimize/_qap.py

```diff
@@ -0,0 +1,780 @@
+        accomplished through trivial negation of the objective function.
+
+    rng : {None, int, `~np.random.RandomState`, `~np.random.Generator`}
+        This parameter defines the object to use for drawing random
+        variates.
+        If `rng` is ``None`` the `~np.random.RandomState` singleton is
+        used.
+        If `rng` is an int, a new ``RandomState`` instance is used,
+        seeded with `rng`.
+        If `rng` is already a ``RandomState`` or ``Generator``
+        instance, then that object is used.
+        Default is None.
+
+    partial_match : 2d-array of integers, optional, (default = None)
+        Allows the user to fix part of the matching between the two
+        matrices. In the literature, a partial match is also known as a
+        "seed".
+
+        Each row of `partial_match` specifies the indices of a pair of
+        corresponding nodes, that is, node ``partial_match[i, 0]`` of `A` is
+        matched to node ``partial_match[i, 1]`` of `B`. Accordingly,
+        ``partial_match`` is an array of size ``(m , 2)``, where ``m`` is
+        not greater than the number of nodes.
+
+    P0 : 2d-array, "barycenter", or "randomized" (default = "barycenter")
+        The initial (guess) permutation matrix or search "position"
+        :math:`P0`.
+
+        :math:`P0` need not be a proper permutation matrix;
+        however, it must have the same shape as `A` and
+        `B`, and it must be doubly stochastic: each of its
+        rows and columns must sum to 1.
+
+        If unspecified or ``"barycenter"``, the non-informative "flat
+        doubly stochastic matrix" :math:`J = 1*1^T/n`, where :math:`n`
+        is the number of nodes and :math:`1` is a :math:`n \times 1`
+        array of ones, is used. This is the "barycenter" of the
+        search space of doubly-stochastic matrices.
+
+        If ``"randomized"``, the algorithm will start from the
+        randomized initial search position :math:`P_0 = (J + K)/2`,
+        where :math:`J` is the "barycenter" and :math:`K` is a random
+        doubly stochastic matrix.
+
+    shuffle_input : bool (default = False)
+        To avoid artificially high or low matching due to inherent
+        sorting of input matrices, gives users the option
+        to shuffle the nodes. Results are then unshuffled so that the
+        returned results correspond with the node order of inputs.
+        Shuffling may cause the algorithm to be non-deterministic,
+        unless a random seed is set or an `rng` option is provided.
+
+    maxiter : int, positive (default = 30)
+        Integer specifying the max number of Franke-Wolfe iterations performed.
+
+    tol : float (default = 0.03)
+        A threshold for the stopping criterion. Franke-Wolfe
+        iteration terminates when the change in search position between
+        iterations is sufficiently small, that is, when the relative Frobenius
+        norm, :math:`||P_{i}-P_{i+1}||_F \over \sqrt{len(P_{i})} \leq tol`,
```

Comment:
```suggestion
        norm, :math:`||P_{i}-P_{i+1}||_F \over \sqrt{n)} \leq tol`,
```
We have already defined `n` above, so I think it is fine to reuse it.

Reply:
If `n` is the number of nodes, and `m` is the number of seeded nodes, then `P0` should be `(n-m) x (n-m)`. I will update the docs to reflect this.

Reply:
My understanding was wrong. Sorry for noise.
No need for a change.